src/checks.rs

```diff
@@ -23,6 +24,7 @@ fn checklist() -> Vec<Box<dyn Check>> {
         Box::new(lowercase_key::LowercaseKeyChecker::default()),
         Box::new(space_character::SpaceCharacterChecker::default()),
         Box::new(unordered_key::UnorderedKeyChecker::default()),
+        Box::new(quote_character::QuoteCharacterChecker::default()),
```

Comment:
Please put this check after `Box::new(lowercase_key::LowercaseKeyChecker::default())`.